### ISSUE TITLE
TextureLoadingGroupMixin:RemoveTexture() missing "texture" parameter

### ISSUE THREAD

**Issue by Meorawr** [Stanzilla/WoWUIBugs#104]
The [TextureLoadingGroupMixin:RemoveTexture](https://github.com/Gethe/wow-ui-source/blob/5669806c19d84e0da9f324f665855b30730f0457/FrameXML/MixinUtil.lua#L309)() function is missing a "texture" parameter, so calling it will result in a possible "table index is nil" error, or doing the wrong thing entirely, as it attempts to assign a nil value to a key based on the `texture` global.

```lua
local TLG = CreateFromMixins(TextureLoadingGroupMixin)
TLG:AddTexture(1);
TLG:RemoveTexture(1);  -- error: "table index is nil" assuming no texture global has leaked
```

**Comment by Meorawr** [comment]
Still present in 1.14.0.

**Comment by Meorawr** [comment]
Fixed in 1.14.0.40179.
